src/System.IO.FileSystem/tests/Performance/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard-Windows_NT;
```

Comment:
I picked windows specific because the Configuration setting was windows specific. I can change it though.